extensions/README.md

```diff
@@ -5,9 +5,9 @@
 - Version: 4.5
 - License: MIT
 
-## DrawNodeEx
+## DrawNodeEx => DrawNodeV2
 - Upstream: https://github.com/axmolengine/axmol
-- Version: 0.9
+- Version: still in progress...
```

Comment:
please just display the actual version, thanks

Reply:
Ok...numbers only